terraform/backend/module/alb.tf

```diff
@@ -1,12 +1,41 @@
+data "aws_region" "current" {}
+data "aws_caller_identity" "current" {}
```

Comment:
these are/should be in data.tf

Reply:
Addressed in https://github.com/codersagainstcovidorg/infra/pull/5/commits/b68be753c6c3b9e2cdc7f1a69b706bfbca0e1203